### PR TITLE
Feat: Set impersonators for API v2 venues/journals

### PR DESCRIPTION
**Pull request by carlosmondra**

### CODE REVIEW

File: openreview/venue/group.py [hunk 1]
```diff
@@ -561,3 +561,9 @@ def create_paper_group(submission):
 
         if create_paper_groups:
             tools.concurrent_requests(create_paper_group, venue.get_submissions(sort='number:asc'), desc='Creating paper groups')
+
+    def set_impersonators(self, impersonators):
```
Comment (melisabok) [inline openreview/venue/group.py:565]: do you mind to add a test where a PC client can impersonate another user?

Comment (carlosmondra) [reply openreview/venue/group.py:565]: Yeah, no problem. I wanted to ask you if you think this is the right place to have this function?

Comment (melisabok) [reply openreview/venue/group.py:565]: I think this place is ok for now.

**Comment by carlosmondra** [comment]
I have added a test

**Review by melisabok — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**